.github/workflows/update_dependencies.yml

```diff
@@ -23,21 +23,13 @@ jobs:
           use-github-cache: false
           use-mathlib-cache: false
 
-      - name: Get sha of branch
-        id: sha
-        run: |
-          SHA="$(git rev-parse --verify origin/update-dependencies-bot-use-only)"
-          echo "sha=$SHA" >> "$GITHUB_OUTPUT"
-
       - name: Get PR and labels
-        if: ${{ steps.sha.outputs.sha }}
         id: PR # all the steps below are skipped if 'ready-to-merge' is in the list of labels found here
         uses: 8BitJonny/gh-get-current-pr@4056877062a1f3b624d5d4c2bedefa9cf51435c9 # 4.0.0
         # TODO: this may not work properly if the same commit is pushed to multiple branches:
         # https://github.com/8BitJonny/gh-get-current-pr/issues/8
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          sha: ${{ steps.sha.outputs.sha }}
```

Comment:
This is no longer necessary with the v4.0.0 of the action; I've also removed the step above (its output is not used anywhere else in this workflow).